icode.properties

```diff
@@ -1,11 +1,19 @@
 category=External Analyzers
-description=Import i-Code CNES results for Fortran(77 & 90) and Shell
+description=Analyze your Fortran(77 & 90) and Shell code with i-Code CNES
 homepageUrl=https://github.com/lequal/sonar-icode-cnes-plugin
 archivedVersions=1.0.2,1.1.0
-publicVersions=
+publicVersions=2.0.1
+
+sonar.pluginKey=icode
 
 defaults.mavenGroupId=fr.cnes.sonarqube.plugins
-defaults.mavenArtifactId=sonaricode
+defaults.mavenArtifactId=sonar-icode-cnes-plugin
+
+2.0.1.description=i-Code CNES is now embedded in the plugin, no more need to install it
+2.0.1.date=2020-02-14
+2.0.1.sqVersions=[7.9,8.1]
```

Comment:
The end range of the current version is typically set to `LATEST`. By setting the end of the range to 8.1, you've already marked yourself with last week's 8.2 release.

Reply:
We had a problem with our cnesreport plugin: we set the sqVersions with [6.7,LATEST] but the plugin is no more compatible with SonarQube 8 and users were able to download it on version 8 before we were aware of the project. 

If you prefer we use LATEST, we can.

Reply:
Not if it's not compatible. :-)
I'm just pointing out that with the compatibility you've set here, the plugin isn't offered to users of SonarQube 8.2